### PR TITLE
Add wikipedia and operator information

### LINKED ISSUE #984: Research needed for: shop/bakery|Anker
#### :thinking: We need help deciding what to do with "shop/bakery|Anker"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/bakery|Anker` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/bakery|Anker`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/bakery|Anker`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #987: Research needed for: shop/bakery|Backwerk
#### :thinking: We need help deciding what to do with "shop/bakery|Backwerk"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/bakery|Backwerk` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/bakery|Backwerk`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/bakery|Backwerk`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1066: Research needed for: shop/bookmaker|Fortuna
#### :thinking: We need help deciding what to do with "shop/bookmaker|Fortuna"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/bookmaker|Fortuna` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/bookmaker|Fortuna`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/bookmaker|Fortuna`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1147: Research needed for: shop/clothes|Brice
#### :thinking: We need help deciding what to do with "shop/clothes|Brice"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/clothes|Brice` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/clothes|Brice`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/clothes|Brice`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1176: Research needed for: shop/clothes|Jules
#### :thinking: We need help deciding what to do with "shop/clothes|Jules"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/clothes|Jules` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/clothes|Jules`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/clothes|Jules`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1214: Research needed for: shop/clothes|Tati
#### :thinking: We need help deciding what to do with "shop/clothes|Tati"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/clothes|Tati` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/clothes|Tati`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/clothes|Tati`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1230: Research needed for: shop/clothes|はるやま
#### :thinking: We need help deciding what to do with "shop/clothes|はるやま"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/clothes|はるやま` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/clothes|はるやま`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/clothes|はるやま`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1236: Research needed for: shop/confectionery|Jamin
#### :thinking: We need help deciding what to do with "shop/confectionery|Jamin"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/confectionery|Jamin` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/confectionery|Jamin`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/confectionery|Jamin`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### PR DESCRIPTION
**Pull request by Adamant36**
Closes #1236, closes #1230, closes #1214, closes #1147, closes #1066, closes #987, closes #984. Also adds some operator Wikipedia entries.

### CODE REVIEW

**Comment by Adamant36** [comment]
@tas50, feel free to review it if you want. I'll wait awhile to merge it.

File: config/canonical.json [hunk 1]
```diff
@@ -16791,6 +16793,8 @@
     "match": ["shop/bakery|BackWerk"],
     "tags": {
       "brand": "Backwerk",
```
Comment (tas50) [inline config/canonical.json]: The W should be capitalized throughout this entry

File: config/canonical.json [hunk 2]
```diff
@@ -20404,6 +20421,8 @@
     "count": 145,
     "tags": {
       "brand": "Jules",
+      "brand:wikidata": "Q25351738",
+      "brand:wikipedia": "en:Joules (clothing)",
```
Comment (tas50) [inline config/canonical.json]: I looked at this one earlier. I don't think this is the right wikipedia article. That's a UK clothing store right above this entry and most of the Jules stores are in France. Different place.

File: config/canonical.json (hunk 1 above)
Comment (Adamant36) [reply config/canonical.json]: Hhhmmm, normally its good to go with established tagging and not create new tags, but in the case of the brand tag there is only 3 of them mapped in lowercase anyway. So I guess it doesn't matter. With the name tag there's 179 in lower case and 56 with a capital letter. So its probably better to keep that one lowercase.

File: config/canonical.json (hunk 2 above)
Comment (Adamant36) [reply config/canonical.json]: Good catch. Your probably right.

**Pull request merged**